### DISCUSSION TITLE
Delivery time ?

### DISCUSSION THREAD

**Discussion by mspbonn** [john30/ebusd#603] [Duplicates]
How long will it currently last from reservation of the the ebus module until delivery ?

**Comment by doofy** [comment]
I placed my reservation on 30.10.2021 and still have not received the adapter. [Docs](https://ebusd.de/meinadapter/) say all reservations until 10.09.2021 have been fullfilled. I guess the global chip shortage is the main problem.

**Reply by doofy** [reply]
See: #554

**Comment by john30** [comment]
duplicate of #554
